Q/Qhull/build_tarballs.jl

```diff
@@ -0,0 +1,54 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "Qhull"
+version = v"2019.1"
+
+# Collection of sources required to build FFMPEG
+sources = [
+    "https://github.com/qhull/qhull/archive/$(version.major).$(version.minor).tar.gz" =>
+    "cf7235b76244595a86b9407b906e3259502b744528318f2178155e5899d6cf9f",
+]
+
+# Bash recipe for building across all platforms
+# TODO: Theora once it's available
+script = raw"""
+# initial setup
+cd $WORKSPACE/srcdir/qhull-*
+
+# begin the build process
+
+cd build
+# Generate makefiles
+cmake -G "Unix Makefiles" .. && cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+# Ensure the config is correct
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
```

Comment:
I'm not familiar with this library, is this double invocation of `cmake` necessary?  I have the feeling the first one is unnecessary.

Reply:
The comments in the build log talk about running `cmake` again. I guess it can't hurt.

Reply:
The reason I did that was because it was suggested in the build instructions for cmake.  I'm not too familliar with it, so I figured I'd just go with it - I can always remove it later.